config/test_utils.go

```diff
@@ -1,5 +1,3 @@
-//go:build testing
```

Comment:
Please keep this tag in this file. This file is only used in unit tests, so we use the tag to exclude this file from release builds.